src/Common.targets

```diff
@@ -12,12 +12,5 @@
 		<!-- Enable ShouldWriteErrorOnInvalidXaml for all the sample and test projects of this solution -->
 		<ShouldWriteErrorOnInvalidXaml>True</ShouldWriteErrorOnInvalidXaml>
 	</PropertyGroup>
-	
-	<Target Name="_UnoCommonFeatureDefines" AfterTargets="PrepareForBuild">
 
-		<PropertyGroup Condition="'$(MSBuildVersion)' &gt;= '15.4'">
-			<DefineConstants>$(DefineConstants);__IOS_11__</DefineConstants>
```

Comment:
Setting `__IOS_11__` unconditionally doesn't sound correct to me. I also found zero references of `__IOS_11__`, so I think it's safe to delete this altogether.

Reply:
It may have been referenced to in apps a very long time ago, but we can remove it now, and it's incorrect anyways.